x-pack/test/functional/services/observability/alerts/common.ts

```diff
@@ -266,6 +295,9 @@ export function ObservabilityAlertsCommonProvider({
     setWorkflowStatusForRow,
```

Comment:
Same comment as above regarding remove workflowStatus leftovers

Reply:
used by other tests... :)

Reply:
Ok I can approve this PR and we could take care of left overs in a follow up PR. Looks like @fkanout already created an issue https://github.com/elastic/kibana/issues/119946